docker/plan.ps1

```diff
@@ -1,12 +1,12 @@
 $pkg_name = "docker"
 $pkg_description = "The Docker Engine"
 $pkg_origin = "core"
-$pkg_version = "17.09.0"
+$pkg_version = "18.09.0"
 $pkg_maintainer = "The Habitat Maintainers <humans@habitat.sh>"
 $pkg_license = @("Apache-2.0")
-$pkg_source = "https://download.docker.com/win/static/stable/x86_64/$pkg_name-$pkg_version-ce.zip"
+$pkg_source = "https://download.docker.com/linux/static/stable/x86_64/${pkg_name}-${pkg_version}.tgz"
```

Comment:
Looks like this got caught in a copy/paste 😄.    I did some digging on this to verify the URL and it looks like docker stopped publishing  windows releases to this location,  the latest package present is `17.09.0` version.  We'll need to do some investigating on this to determine what the source should be.

Reply:
Yeah :)
I looked around, but it seems you have to log into an account to download the windows ce edition [here](https://hub.docker.com/editions/community/docker-ce-desktop-windows). If there's a straight link to an archive, I'll put it in the `ps1`.

Reply:
I think in this case we can leave the windows out of the update.